src/Modules/Internationalisation/Domain/Twig/ForkIntlExtension.php

```diff
@@ -246,8 +252,10 @@ private function createIntlDateFormatter(
     }
 
     /**
+     * @codingStandardsIgnoreStart
```

Comment:
I don't know why it is needed, but TBH it adds so much clutter, is this really needed?

Reply:
fixed c3193626f6706891e887ac7d8be697bbe998fc92